scattr/workflow/Snakefile

```diff
@@ -12,10 +12,24 @@ configfile: "config/snakebids.yml"
 
 
 # writes inputs_config.yml and updates config dict
-inputs = generate_inputs(
+inputs_t1w = generate_inputs(
     bids_dir=config["bids_dir"],
     pybids_inputs=config["pybids_inputs"],
     pybids_config=["bids", "derivatives"],
+    pybidsdb_dir=config.get("pybidsdb_dir"),
+    pybidsdb_reset=config.get("pybidsdb_reset"),
+    derivatives=config["derivatives"],
+    participant_label=config["participant_label"],
+    exclude_participant_label=config["exclude_participant_label"],
+)
+inputs_dwi = generate_inputs(
+    bids_dir=config["dwi_dir"] if config["dwi_dir"] else config["bids_dir"],
+    pybids_inputs=config["pybids_inputs_dwi"],
+    pybids_config=["bids", "derivatives"],
+    pybidsdb_dir=config.get("pybidsdb_dwi_dir")
```

Comment:
This can just be `config.get("pybidsdb_dwi_dir", config.get("pybidsdb_dir"))`. Now that I think about it we probably want this to be `config.get("pybidsdb_dwi_dir") if config["dwi_dir"] else config.get("pybidsdb_dir")` to handle the case where someone wants to use a `dwi_dir` but doesn't want to use a pybids db for that dataset.